doc/how_to/custom_components/reactive_html/reactive_html_layout.md

```diff
@@ -0,0 +1,292 @@
+# Create Layouts With ReactiveHTML
+
+In this guide you will learn how to build custom layouts using HTML and `ReactiveHTML`.
+
+If you are not familiar with HTML then the [W3 HTML School](https://www.w3schools.com/html/default.asp)
+is a good resource to learn from. You can also ask ChatGPT for help. It can often provide you with
+a good HTML starting point that you can fine tune.
+
+## A layout of a single object
+
+You can layout a single object as follows.
+
+```{pyodide}
+import panel as pn
+import param
+
+pn.extension()
+
+class LayoutSingleObject(pn.reactive.ReactiveHTML):
+    object = param.Parameter()
+
+    _ignored_refs = ("object",)
+
+    _template = """
+    <div style="height:100%;width:100%">
+        <h1>Temperature</h1>
+        <h2>A measurement from the sensor</h2>
+        <div id="object" class="styled-block">${object}</div>
+    </div>
+"""
+
+dial = pn.widgets.Dial(
+    name="°C", value=37, format="{value}", colors=[(0.40, "green"), (1, "red")], bounds=(0, 100),
+)
+LayoutSingleObject(
+    object=dial, name="Temperature", description="A measurement from the sensor", styles={"border": "2px solid lightgray"}, width=500
+).servable()
+```
+
+Please notice
+
+- We define the HTML layout in the `_template` attribute.
+- In the `_template` attribute we define an outer *container* `div` with a `height` and `width` of
+`100%`. This will make your `_template` responsive, i.e. (re-)size it self to the `height` and
+`width` of your `ReactiveHTML` component.
+- We can refer to the parameter `object` in the `_template` via the *template parameter* `${object}`.
+  - We must give the `div` element holding the `${object}` an `id`. If we do not do this an exception
+  will be raised. The `id` can be any value, for example `id="my-object"`.
+- We call our *object* parameter `object` to be consistent with our built in layouts. But the
+parameter can be called anything. For example `value`, `dial` or `temperature`.
+- We add the `border` in the `styles` parameter so that we can better see how the `_template` layes
+it self out inside the `ReactiveHTML` component. This can be useful for development.
+
+## A Layout of multiple objects
+
+```{pyodide}
+import panel as pn
+import param
+
+pn.extension()
+
+class LayoutMultipleValues(pn.reactive.ReactiveHTML):
+    object1 = param.Parameter()
+    object2 = param.Parameter()
+
+    _ignored_refs = ("object1", "object2")
+
+    _template = """
+    <style>
```

Comment:
Instead of inlining the CSS I need guidance on the best practice for adding CSS @philippjfr . I can't see how this should be done.

I can of course add CSS via the `stylesheets` parameter. But that will leave a risk that it gets overridden by users if the set the `stylesheets` parameter on construction or after.

Reply:
Can you try using `_stylesheets`? It was designed for other components but **should** work with ReactiveHTML as well.